Automattic/jetpack
# Enable My Jetpack for disconnected sites

#### Changes proposed in this Pull Request:

* Enables My Jetpack for disconnecetd sites

#### Jetpack product discussion


p1647980905490769-slack-C02TQF5VAJD

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* On a new disconnected site
* Make sure My Jetpack shows up
* Try to use all its features and see if they work
